template/index.html

```diff
@@ -7,7 +7,7 @@
     <title>{{ name }}</title>
     <link rel="icon" type="image/png" sizes="32x32" href="<%= htmlWebpackPlugin.files.publicPath %>static/img/icons/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="<%= htmlWebpackPlugin.files.publicPath %>static/img/icons/favicon-16x16.png">
-    <!--[if IE]><link rel="shortcut icon" href="/static/img/icons/favicon.ico"><![endif]-->
+    <!--[if IE]><link rel="shortcut icon" href="<%= htmlWebpackPlugin.files.publicPath %>static/img/icons/favicon.ico"><![endif]-->
```

Comment:
This seems like a reasonable change to make independent from https://github.com/vuejs-templates/pwa/pull/82/files#r142584472. If you wanted to modify the PR to just include this change, and leave out the other change for now, that would be mergeable.